app/views/foreman_wreckingball/hosts/schedule_remediate.html.erb

```diff
@@ -1,35 +1,52 @@
-<% title _('Remediate %s') % @host %>
+<% title _('Remediate') %>
 <% javascript 'foreman_tasks/trigger_form' %>
 <% stylesheet 'foreman_tasks/trigger_form' %>
 
+<% if @statuses.any? %>
+  <% if @statuses.group_by(&:class).map { |i| i.first}.select { |i| i.try(:dangerous_remediate?) }.any? %>
+    <%= alert(:text => _('This will cause a service interruption.'), :class => 'alert-warning', :close => false) %>
+  <% end %>
 
-<% if @status.class.respond_to?(:dangerous_remediate?) && @status.class.dangerous_remediate? -%>
-<%= alert(:text => _('This will cause a service interruption.'), :class => 'alert-warning', :close => false) %>
-<% end -%>
+  <%= _('%s hosts selected for remedate' % @statuses.count) %>
```

Comment:
Typo: `remediate`, but i think it should be `for remediation.`

@leewaa: Can you help?

Reply:
Also: Please use `n_("One", "Two", number)` so plural forms are matched correctly.